source/administration/mmctl-cli-tool.rst

```diff
@@ -0,0 +1,2318 @@
+
+.. code-block:: sh
+
+  --format string   the format of the command output [plain, json] (default "plain")
+
+
+mmctl user email
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**Description**
+
+  Change the email address associated with a user.
+
+**Format**
+
+.. code-block:: sh
+
+    mmctl user email [user] [new email] [flags]
+
+**Examples**
+
+.. code-block:: sh
+
+  user email test user@example.com
+  user activate username
+
+**Options**
+
+.. code-block:: sh
+
+    -h, --help       help for email
+
+
+**Options Inherited from Parent Commands**
+
+.. code-block:: sh
+
+  --format string   the format of the command output [plain, json] (default "plain")
+
+mmctl user invite
+^^^^^^^^^^^^^^^^^^
+
+**Description**
+
+  Send an email invite to a user, to join a team. You can invite a user to multiple teams by listing
+  them. You can specify teams by name or ID.
+
+**Format**
+
+.. code-block:: sh
+
+    mmctl user invite [email] [teams] [flags]
+
+**Examples**
+
+.. code-block:: sh
+
+  user invite user@example.com myteam
+  user invite user@example.com myteam1 myteam2
+
+**Options**
+
+.. code-block:: sh
+
+    -h, --help       help for invite
+
+**Options Inherited from Parent Commands**
+
+.. code-block:: sh
+
+  --format string   the format of the command output [plain, json] (default "plain")
+
+mmctl user reset_password
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**Description**
+
+  Send users an email to reset their password.
+
+**Format**
+
+.. code-block:: sh
+
+    mmctl user reset_password [users] [flags]
+
+**Examples**
+
+.. code-block:: sh
+
+  user reset_password user@example.com
+
+**Options**
+
+.. code-block:: sh
+
+    -h, --help       help for reset_password
+
+
+**Options Inherited from Parent Commands**
+
+.. code-block:: sh
+
+  --format string   the format of the command output [plain, json] (default "plain")
+
+mmctl user resetmfa
+^^^^^^^^^^^^^^^^^^^^
+
+**Description**
+
+  Turn off multi-factor authentication for a user. If MFA enforcement is enabled, the
+  user will be forced to re-enable MFA as soon as they login.
+
+**Format**
+
+.. code-block:: sh
+
+    mmctl user resetmfa [users] [flags]
+
+**Examples**
+
+.. code-block:: sh
+
+    user resetmfa user@example.com
+
+**Options**
+
+.. code-block:: sh
+
+    -h, --help       help for resetmfa
+
+
+**Options Inherited from Parent Commands**
+
+.. code-block:: sh
+
+  --format string   the format of the command output [plain, json] (default "plain")
+
+mmctl user search
+^^^^^^^^^^^^^^^^^^
+
+**Description**
+
+  Search for users based on username, email, or user ID.
+
+**Format**
+
+.. code-block:: sh
+
+    mmctl user search [users] [flags]
+
+**Examples**
+
+.. code-block:: sh
+
+    user search user1@mail.com user2@mail.com
+
+**Options**
+
+.. code-block:: sh
+
+    -h, --help       help for search
+
+
+**Options Inherited from Parent Commands**
+
+.. code-block:: sh
+
+  --format string   the format of the command output [plain, json] (default "plain")
+
+mmctl version
```

Comment:
Few sections here are incomplete. May be intentional

Reply:
Removed.